lib/readline.js

```diff
@@ -56,7 +57,7 @@ function Interface(input, output, completer, terminal) {
   }
 
   if (completer && typeof completer !== 'function') {
-    throw new TypeError('Argument "completer" must be a function');
+    throw new errors.TypeError('ERR_INVALID_CALLBACK');
```

Comment:
the original message contains useful information(`Argument "completer"`), this looks like a `ERR_INVALID_OPT_VALUE` to me (refs: https://github.com/nodejs/node/pull/11300/files), passing the `completer` as an actual argument is not documented(refs: [readline.createInterface()](https://nodejs.org/api/readline.html#readline_readline_createinterface_options)) so it doesn't fit the description of a Node.js style "callback" I think.